src/apps/relay/prom_server.c

```diff
@@ -129,15 +129,17 @@ void prom_set_finished_traffic(const char* realm, const char* user, unsigned lon
   }
 }
 
-void prom_inc_allocation(void) {
+void prom_inc_allocation(SOCKET_TYPE type) {
   if (turn_params.prometheus == 1){
-    prom_gauge_inc(turn_total_allocations, NULL);
+    prom_gauge_inc(turn_total_allocations, (const char*[]) {"all"});
```

Comment:
Not required
We only need to reports the type and the aggregation will be done in prometheus query. For example, this explicit query for all types
```
turn_total_allocations{type~=".*"}
```
If we report both "all" and "udp" then every allocation will be counted twice (in that query)

Reply:
Actually, it's a different number. "all" should count all independent of the type and "udp" should be only udp. During my tests I saw at least "udp" and "tcp". Still, I totally agree that prometheus or the tool to draw graphs can then be used to calculate "all". If you prefer, I can remove the "all" field again.